presentations/control-flow/11a.rs

```diff
@@ -1,9 +1,13 @@
-'outer: for i in 0..10 {
-    loop {
-        if i < 5 {
-            continue 'outer;
-        } else {
-            break 'outer;
+fn main() {
+    'outer: for i in 0..10 {
+        for j in 0..5 {
```

Comment:
I just ran through this today and people suggested also labeling the inner loop `'inner` to highlight what we break and where we jump to. Wdyt?

Reply:
I'm usually a fan of explicit over implicit 🙃